pkg/printer/printer.go

```diff
@@ -242,17 +260,30 @@ func (p *Printer) WriteProtoFlow(res *observerpb.GetFlowsResponse) error {
 			return fmt.Errorf("failed to write out packet: %v", err)
 		}
 	case CompactOutput:
+		var err error
 		src, dst := p.GetHostNames(f)
-		_, err := fmt.Fprintf(p.opts.w,
-			"%s [%s]: %s -> %s %s %s (%s)\n",
-			fmtTimestamp(f.GetTime()),
-			f.GetNodeName(),
-			src,
-			dst,
-			GetFlowType(f),
-			f.GetVerdict().String(),
-			f.GetSummary(),
-		)
+
+		if !p.opts.nodeName {
```

Comment:
this can be one statement i think. we can make the whole `[node-name-here]` a value, and then pass in `""` in the no node name case

Reply:
That's a good idea. I will change it sometime tonight